be/src/exprs/function_call_expr.cpp

```diff
@@ -167,6 +167,13 @@ StatusOr<ColumnPtr> VectorizedFunctionCallExpr::evaluate_checked(starrocks::Expr
         result = _fn_desc->scalar_function(fn_ctx, args);
     }
     RETURN_IF_ERROR(result);
+    if (_fn_desc->check_overflow) {
+        std::string err_msg;
+        if (UNLIKELY(result.value()->capacity_limit_reached(&err_msg))) {
```

Comment:
can we split the chunk to avoid such overflow?

Reply:
Splitting chunks requires a mature plan, which is relatively complicated. Currently, this is done to ensure that there will be no crashes and incorrect results.